app/Config/Filters.php

```diff
@@ -15,11 +15,16 @@ class Filters extends BaseConfig
 	// Always applied before every request
 	public $globals = [
 		'before' => [
+			'toolbar' => [
+				'except' => 'api/*',
```

Comment:
Please remove the exceptions. They make an assumption about projects that the framework shouldn't make.